### DISCUSSION TITLE
is there any way I can load the data from database during initialization?

### DISCUSSION THREAD

**Discussion by Shashi-GS** [vinissimus/next-translate#573] [Q&A]
Hi,
  is there any way I can load the data i18n initialization? during "yarn run build" it has to collect the locale data from the Firebase database. is there any property I can use for that?

--
Thanks,
Shashi

**Comment by aralroca** [comment]
@Shashi-GS the `loadLocaleFrom` is to load the namespaces during build time, in case of `getStaticProps`, or run-time in case of `getServerSideProps`.

In `i18n.js` file:

```js
{
 ...restOfConfig,
  loadLocaleFrom: (lang, ns) =>
      fetch(`https://cdn.mylocales.com/${lang}/${ns}`).then((res) => res.json()),
}
```
